src/aat/resources/cucumber/questionrounds.feature

```diff
@@ -5,12 +5,16 @@ Feature: Question Round Logic
     And a standard question
     Given the question round is ' "1" '
     When the post request is sent to create the question
-    And the response code is 200
+    Then the response code is 200
+    When the put request is sent to issue the question round ' "1" '
```

Comment:
In addition check that an event history is created for each question that has been issued.